### PR TITLE
Readme patch 1 to implement 1

### PR DESCRIPTION
**Pull request by schettn**
**Checklist:**

 
 - [x] Have you added an explanation of what your changes do and why you'd like them to be included?
 - [x] Have you updated or added documentation for the change?
 - [x] Have you tested your changes with successful results?

**Type of Changes:**

 
 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [ ] New feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to change)

**What is the current behavior? (link to any open issues here)**
- Currently there is no readme which represents the snek-client. #4

**What is the new behavior (if this is a feature change)?**
- Now there is a readme which represents the snek-client. #4

### CODE REVIEW

**Review by Aichnerc — APPROVED** [review]
LGTM 🍷

File: README.md [hunk 1]
```diff
@@ -76,36 +70,26 @@ Get updates on our development and chat/talk with the project maintainers and co
...
 We do not have any external contributors yet, but if you want your name to be here, feel free
 to [contribute to our project](#contributing).
 
-## [](#vault)GitHub Arctic Vault
-This repository is ready for the GitHub Archive Program and therefore meets our GitHub guidelines and 2020 code style.
-<a href="https://archiveprogram.github.com/" target="_blank" rel="noopener noreferrer">
-  <img src="https://www.aichner-christian.com/img/logo/code_vault.jpg" style="margin-top: 0.5rem" alt="Mars Time Logo" height="35">
-</a>
-
 ## [](#copyright-and-license)Copyright and license
```
Comment (pinterid) [inline README.md:93]: The license is wrong.

**Review by pinterid — CHANGES_REQUESTED** [review]
Looks good after 1 change.

**Review by pinterid — APPROVED** [review]
Looks good.

**Pull request merged**